### ISSUE TITLE
Error During Run

### ISSUE THREAD

**Issue by hsachdevah** [jleni/wallet-decrypt#1]
Hey jleni,

Thanks for your contribution, can you please help with this error below. I'm using ubuntu to run your code on python2.7

```
Traceback (most recent call last):
  File "wallet-decrypt.py", line 56, in <module>
    main()
  File "wallet-decrypt.py", line 47, in main
    w = get_wallet(args.filename, args.password)
  File "wallet-decrypt.py", line 34, in get_wallet
    pbdata = padded_plain[:-pad_len]
TypeError: bad operand type for unary -: 'str'
```
Is it a known issue? When you say use filename as argument, do you mean backup file generated from bitcoin android wallet?

**Comment by hsachdevah** [comment]
No error after using python3. But blank output

**Comment by jleni** [comment]
Thanks for reporting this. I will have a look later today. I didn't have those problems but I didn't expect a lot of people to be interested. I will check what is going on.

**Comment by jleni** [comment]
This issue is fixed in PR #3
